Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Script/HandInteractionPan.cs

```diff
@@ -469,16 +468,19 @@ public void OnTouchStarted(HandTrackingInputEventData eventData)
             if (touchingSource == null)
             {
                 currentPointer = eventData.InputSource.Pointers[0];
-                currentController = eventData.Controller;
+                currentController = currentPointer.Controller;
```

Comment:
Is there still a need to track currentController?

Reply:
agree about redundancy. I removed currentPointer though- because controller seems more useful